wait-for-postgres.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+while !</dev/tcp/${MF_METADATA_DB_HOST}/${MF_METADATA_DB_PORT}; do
+    sleep 1;
```

Comment:
there probably should be an upper limit on how long we wait. In production we try 5 times by default with a 1 second sleep in between each attempt

Reply:
Good point. Added these as configurable env vars now (in case values need to be tuned for github test runners or other environments):

- `POSTGRES_WAIT_MAX_RETRIES` default 5
- `POSTGRES_WAIT_SLEEP_SECONDS` default 1